variables.tf

```diff
@@ -397,3 +397,10 @@ variable "cluster_parameters" {
   description = "A list of cluster parameter objects"
   default     = []
 }
+
+variable "enable_global_cluster" {
+  type        = bool
+  description = "Set this variable to `true` if DB Cluster is going to be part of a Global Cluster."
+  default     = false
+}
```

Comment:
How is this variable set to `true` in the example in `examples/global-aurora-mysql/main.tf`?

Reply:
Example is actively being worked on. It will be used soon